documentation/book/snip-kafka-versions.adoc

```diff
@@ -6,8 +6,6 @@
 [options="header"]
 |=================
 |Kafka version |Interbroker protocol version |Log message format version| Zookeeper version
-|2.1.0 |2.1 |2.1 |3.4.13
 |2.1.1 |2.1 |2.1 |3.4.13
-|2.2.0 |2.2 |2.2 |3.4.13
```

Comment:
I think this file needs to be reverted. It is generated via `make docu_versions`, based on the `kafka-versions` file.

Reply:
Reverted